tests/testthat/test_changeValLabels.R

```diff
@@ -1,21 +1,24 @@
 
-# dfSAV <- import_spss(file = "tests/testthat/helper_spss_missings.sav")
-dfSAV <- import_spss(file = "helper_spss_missings.sav")
+dfSAV <- import_spss(file = test_path("helper_spss_missings.sav"))
 
 dfUn <- import_DF(data.frame(v1 = 1, v2 = 2))
 
+test_that("changeValLabel input validation", {
+  expect_error(changeValLabels(dfSAV, varName = c("VAR1"), value = 1:2, valLabel = "test label"),
+               "value and valLabel are not of identical length.")
+  expect_error(changeValLabels(dfSAV, varName = c("VAR4"), value = 1, valLabel = "test label"))

```

Comment:
Nit: Maybe, the expectation could be added? 🤔

```r
The following 'varName' are not variables in the GADSdat: VAR4
```

Reply:
Sounds a bit strange due to singular and plural.

Maybe, this reformulation works?
`The following variables in 'varName' are not ...`

Reply:
I have added the suggested expectation, although I am always a bit hesitant regarding testing exact error messages that are produced in another function. For instance, changing the error message in `check_vars_in_GADSdat()` as suggested leads to errors in 10 tests across `eatGADS` (and only three of them are about `check_vars_in_GADSdat()` itself. 

In general, I think it would be a good idea to do a complete overhaul regarding the input validation; for now, beyond the scope of this PR.